storm-server/src/main/java/org/apache/storm/scheduler/TopologyDetails.java

```diff
@@ -514,8 +514,13 @@ private void initConfigs() {
         this.topologyPriority =
             ObjectReader.getInt(topologyConf.get(Config.TOPOLOGY_PRIORITY), null);
 
-        assert this.topologyWorkerMaxHeapSize != null;
-        assert this.topologyPriority != null;
+        // Fails in storm-core: org.apache.storm.scheduler-test / testname: test-cluster
```

Comment:
I would leave these asserts here and file a JIRA to address fixing the test and replacing these asserts specifically.

Reply:
https://issues.apache.org/jira/browse/STORM-3732

Reply:
Cannot reproduce clojure test failure, closing https://issues.apache.org/jira/browse/STORM-3732

Reply:
Upon further investigation:
    - assert (original code) does not cause Clojure test failure - because assert is not active at runtime
    - replacement of assert - using AssertionError - (new code) will cause Clojure test failure because it is active at runtime